less/settings.py

```diff
@@ -32,63 +32,63 @@
 # Application definition
 
 INSTALLED_APPS = [
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    #3rd party:
-    'crispy_forms',
-    #local:
-    'less_users.apps.LessUsersConfig',
-    'challenges.apps.ChallengesConfig',
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    # 3rd party:
+    "crispy_forms",
+    "debug_toolbar",
```

Comment:
debug toolbar should be enabled only for dev environment

Reply:
Widziałam gdzieś opcję, żeby stworzyć drugi plik test_settings.py, tam zaimportować wszystko z settings.py i dodać elementy związane z django_debug_toolbar: 

```
DEBUG = env.bool("DJANGO_DEBUG", default=False)

if DEBUG:
    INTERNAL_IPS = ["127.0.0.1",]
    INSTALLED_APPS.append("debug_toolbar")  # noqa
    MIDDLEWARE.append("debug_toolbar.middleware.DebugToolbarMiddleware")  # noqa
```
(Przy okazji sprawdziłam co znaczy #noqa) 
nie jestem tylko pewna co ma robić ta pierwsza linijka (tzn. sprawdzić czy w env jest podana wartość DJANGO_DEBUG...

Reply:
to może dev_settings.py? 
Powinnam go dodać na tym samym poziomie co settings.py i ustawić tak, żeby Django korzystało z niego (tego dev_settings) gdy DEBUG = True, a w przeciwnym razie ze zwykłych settings.py?

Reply:
OK